user-registration-app/src/app/models/registration.ts

```diff
@@ -120,7 +114,7 @@ export class Registration {
     registration.expectations = value.expectations;
     registration.veteran = value.veteran;
     registration.submitted = value.submitted;
-    registration.hackathon = this.parseHackathon(registration);
+    registration.hackathon = Hackathon.parseJSON(registration).uid;
```

Comment:
hackathon is supposed to take the string uid, but for some reason it was getting set to the entire hackathon object